builder/bundles.go

```diff
@@ -877,7 +878,40 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func fixOSRelease(filename, version string) error {
+func fixOSRelease(b *Builder, filename, version string) error {
+
+	//Replace the default os-release file if customized os-release file found
+	path := b.Config.Mixer.OSReleasePath
+	if path != "" {
+
+		// Validate file path
+		_, err := os.Stat(path)
+		if err != nil {
+			return err
+		}
+
+		fSrc, err := os.OpenFile(path, os.O_RDONLY, 0666)
+		if err != nil {
+			return err
+		}
+
+		fDst, err := os.OpenFile(filename, os.O_RDWR, 0666)
+		if err != nil {
+			return err
+		}
+
+		//Overwrite the os-release file
+		_, err = io.Copy(fDst, fSrc)
+		if err != nil {
+			return err
+		}
+
+		//Close file descriptor to make sure the copy operation is completed.
+		_ = fSrc.Close()
+		_ = fDst.Close()
```

Comment:
Those are not needed if you use `defer` clause.

Reply:
No issue anymore since doesn't use copy operation.